src/coq_lib.ml

```diff
@@ -0,0 +1,55 @@
+(* This file is licensed under The MIT License *)
+(* (c) MINES ParisTech 2018-2019               *)
+(*     Written by: Emilio Jesús Gallego Arias  *)
+
+open! Stdune
+
+type t =
+  { name : Coq_lib_name.t
+  ; wrapper : string
+  ; src_root : Path.t
+  ; obj_root : Path.t
```

Comment:
Sorry to be picky, but perhaps this should be in `obj_dir`? Or is possible to have more than one directory for artifacts in a coq lib? In OCaml, we can have private modules which requires us to make this separation. Doesn't seem like it's the case for coq though.

Reply:
What do you mean @rgrinberg ? I named it root after your advice indeed as qualified libs do provide qualified object dirs in the current version of Coq.

p.s: don't bother pushing to the PR, my current version has a very large delta, in particular it doesn't use `Lib_name.t` anymore.

Reply:
I see. Well ignore my comment then. I thought the situation was analogous to OCaml where there's only one obj dir (ignoring private modules), but many source directories.